src/components/taskDetails/index.tsx

```diff
@@ -96,9 +92,18 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
     function handleChange(
         event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
     ) {
+        const { name, value } = event.target;
+
+        if (name === 'dependsOn') {
+            const updatedDependencies = value
+                .split(',')
+                .map((taskId) => taskId.trim());
+            setUpdatedDependencies(updatedDependencies);
+        }
```

Comment:
You have used this twice - the logic to split and trim

Reply:
done